src/authentication/UserAuthenticator.js

```diff
@@ -13,6 +13,7 @@ const {
  } = AUTH_ERROR_CODES;
 
 const AUTH_ENDPOINT = '/sync/v3/user';
+const CONNECTION_TIMEOUT_PERIOD = 10000; // 10 second timeout for authenticating connection
```

Comment:
I like to use 10 * 1000 to make it obvious that it is milliseconds, clean to change and avoid a potential mistake